### PR TITLE
Fixed #34318 -- Added release note for 4bfe8c0eec835b8eaffcda7dc1e3b203751a790a.

### PR DESCRIPTION
**Pull request by mackerel711**
About https://code.djangoproject.com/ticket/34318#comment:11

### CODE REVIEW

**Review by emarhin — APPROVED** [review]

File: docs/releases/4.1.txt [hunk 1]
```diff
@@ -289,6 +289,9 @@ Internationalization
 * The :func:`~django.conf.urls.i18n.i18n_patterns` function now supports
   languages with both scripts and regions.
 
+* The :djadmin:`makemessages` commands, :file:`.po` files will not change if there is
+  no change in translations. Formerly POT-Creation-Date was always updated.
```
Comment (felixxm) [inline docs/releases/4.1.txt:293]: This should be moved to the [_"Backwards incompatible changes in 4.1" -> "Miscellaneous"_](https://docs.djangoproject.com/en/4.1/releases/4.1/#miscellaneous).

Comment (felixxm) [inline docs/releases/4.1.txt:293]:
```suggestion
* The :djadmin:`makemessages` command no longer changes :file:`.po` files when
  up to date. In older versions, ``POT-Creation-Date`` was always updated.
```

**Review by felixxm — COMMENTED** [review]
@mackerel711 Thanks :+1: I left comments.

**Comment by mackerel711** [comment]
I appreciate to give some comments. This is my first time to contribute to oss!
I already fixed as  @felixxm 's comments.

**Comment by felixxm** [comment]
@mackerel711 Thanks :+1: Welcome aboard :boat:

**Pull request merged**